src/components/TitleBar.ts

```diff
@@ -53,7 +53,9 @@ const TitleBar = ({
             Link,
             {
               'aria-label': 'Close',
-              style: { marginLeft: '2rem' },
+              style: {
+                /* marginLeft: '2rem' */
```

Comment:
this was looking kinda weird.

Reply:
A bit wary of style changes in a PR for adding a test...

Could you add before/after screenshots for this?

Reply:
This was my mistake - an early attempt to fix the wrong problem :D Reverted.